api/handler.py

```diff
@@ -13,25 +14,22 @@ def make_conn():
     conn = None
     try:
         conn = psycopg2.connect(
-            f"dbname={DB_NAME} user={DB_USER} host={DB_HOST} password={DB_PASS}"
+            f"dbname={DB_NAME} user={DB_USER} host={DB_HOST} password={DB_PASS}  options='-c search_path={DB_SCHEMA}'"
         )
     except:
         print("I'm unable to connect to the database")
     return conn
 
 
 def postLocation(event, context):
-    # body = {
-    #     "message": "POST /location endpoint logic. Your function executed successfully!",
-    #     "input": event,
-    # }
-
-    # response = {"statusCode": 200, "body": json.dumps(body)}
+    payload = json.loads(event["body"])
+    point_str = f"'POINT({payload['longitude']} {payload['latitude']})'"
```

Comment:
I really like the previous version where you included the SRID as well

Reply:
yeah will add it, just wanted to get a working version asap.

Reply:
Done.